content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -1,17 +1,19 @@
 ---
 reviewers:
-- jsafrane
 - saad-ali
 - thockin
 - msau42
+- jingxu97
+- xing-yang
+- yuxiangqian
 title: Volume Snapshots
 content_template: templates/concept
 weight: 20
 ---
 
 {{% capture overview %}}
 
-This document describes the current state of `VolumeSnapshots` in Kubernetes. Familiarity with [persistent volumes](/docs/concepts/storage/persistent-volumes/) is suggested.
+This document describes the current state of `VolumeSnapshots` in Kubernetes. Familiarity with [persistent volumes](/docs/concepts/storage/persistent-volumes/) is suggested. `VolumeSnapshot` is introduced as an alpha feature in Kubernetes 1.12 and is promoted to beta in 1.17.
```

Comment:
If you use a feature state shortcode, this could be:
```suggestion
In Kubernetes, a _VolumeSnapshot_ represents a snapshot of a storage volume. This document assumes that you are already familiar with Kubernetes [persistent volumes](/docs/concepts/storage/persistent-volumes/).
```